examples/index/index.css

```diff
@@ -13,3 +13,9 @@ jw-editable {
     padding: 10px;
     height: 100vh;
 }
+
+jw-editable .drop-cap {
```

Comment:
?

Reply:
Since this PR is a wip, I'm not sure it's ready for review, so I'll stop here.

Reply:
This is for the sake of example. The example page's first letter has a span with that class so it looks like the first letter in a chapter of a book.

Reply:
It's a wip because we need to look into it together so I wouldn't stop the review quite yet.